rpcs3/rpcs3qt/breakpoint_handler.cpp

```diff
@@ -9,14 +9,20 @@ bool breakpoint_handler::HasBreakpoint(u32 loc) const
 
 bool breakpoint_handler::AddBreakpoint(u32 loc)
 {
-	m_breakpoints.insert(loc);
-	ppu_breakpoint(loc, true);
-	return true;
+	if (!ppu_breakpoint(loc, true))
+	{
+		return false;
+	}
+
+	return m_breakpoints.insert(loc).second;
 }
 
 bool breakpoint_handler::RemoveBreakpoint(u32 loc)
 {
-	m_breakpoints.erase(loc);
-	ppu_breakpoint(loc, false);
-	return true;
+	if (!ppu_breakpoint(loc, false))
+	{
+		return false;
```

Comment:
Shouldn't this be the same issue as before with the GUI part? You may end up with a dirty breakpoint list

Reply:
Okay, changed.